src/components/base/NavPage.svelte

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
-    import { applyTheme, themeStore } from '@store/theme';
+       import { themeStore, applyTheme } from "@lib/theme";
```

Comment:
You get a chance to check your prettier setup? It appears that it's not working. 
I'm only harping on it because if devs have different code styles and such, it makes future diffs way bigger than they need to be, taking the focus away from the important bits